src/components/calculator.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/forbid-prop-types */
+/* eslint-disable react/no-array-index-key */
```

Comment:
- [ ] it is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉

Reply:
yeap, but when I enable it, It doesn't allow me to use arrays as PropTypes, same for index as key.